ui/webui/test/check-basic

```diff
@@ -64,5 +64,17 @@ class TestBasic(MachineCase):
         # Ensure that the 'actual' UI process is running/
         self.assertIn("/usr/libexec/webui-desktop", m.execute("ps aux"))
 
+class TestQuit(MachineCase):
+    MachineCase.machine_class = VirtInstallMachine
+
+    def testQuitInstallation(self):
+        b = self.browser
+        m = self.machine
+        i = Installer(b)
+
+        i.open()
+        b.click("button:contains(Quit)")
```

Comment:
Will this work with translations?

Reply:
If it did change the language, no, it would not work.

That's why it better to add ids, in the react elements, and use the id selectors.

Reply:
Ah! Sorry. The button does not have an id because I didn't think I'd need one...